lib/diagnostic.js

```diff
@@ -32,15 +36,21 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.update(members:[
+  "house": "Arryn", "house": "John Stark"
+  "house": "Stark", "house": "Ned Stark"
 
+])
 // your code ends here
 
-
 // Question 3
 //
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
+db.houses.deleteOne({motto:"As High as Honor"})
```

Comment:
This will delete the entire document.  We just want to delete the key from the document.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```